frontend/src/metabase-types/api/mocks/actions.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  WritebackQueryAction,
+  WritebackImplicitQueryAction,
+} from "metabase-types/api";
+import { createMockNativeDatasetQuery } from "./query";
+
+export const createMockQueryAction = ({
+  dataset_query = createMockNativeDatasetQuery(),
+  ...opts
+}: Partial<WritebackQueryAction> = {}): WritebackQueryAction => {
+  return {
+    id: 1,
+    dataset_query,
+    name: "Query Action Mock",
+    description: null,
+    model_id: 1,
+    parameters: [],
+    "updated-at": new Date().toISOString(),
+    "created-at": new Date().toISOString(),
+    ...opts,
+    type: "query",
+  };
+};
+
+export const createMockImplicitQueryAction = (
+  options: Partial<WritebackImplicitQueryAction>,
+): WritebackImplicitQueryAction => ({
+  id: 1,
+  kind: "row/create",
+  name: "",
+  description: "",
+  model_id: 1,
+  "updated-at": new Date().toISOString(),
+  "created-at": new Date().toISOString(),
+  parameters: [
```

Comment:
There should be a helper for creating mock parameters. Another thought - I think that providing an array with 2 specific parameters here might make it harder to follow the test. Maybe leave it empty and pass required parameters when needed?